packages/nx-playwright/src/executors/playwright-executor/executor.ts

```diff
@@ -12,8 +12,12 @@ export default async function executor(
 
   const success = await Promise.resolve()
     .then(async () => {
+      const headedOption = options.headed === true ? '--headed' : '';
```

Comment:
we can start with these options but as a follow-up work, we can allow all possible options